sass/index.scss

```diff
@@ -0,0 +1,35 @@
+.presentation {
+  text-align: center;
+  margin-bottom: 4rem;
+}
+
+.screenshots {
+  display: flex;
+  flex-direction: row;
+
+  img {
+    margin: 0 0.5em;
+    height: 30em;
+  }
+
+  &-wrapper {
+    overflow-x: auto;
+    &-wrapper {
+      width: 75vw;
+      position: relative;
+      left: calc(-38vw + 50%);
+      display: flex;
+      justify-content: center;
+      margin-bottom: 4em;
```

Comment:
```suggestion
      margin-bottom: 4em;
      
      @media (max-width: 800px) {
        width: 100%;
        left: 0;
      }
```

Reply:
There is a lot of white space on mobile, maybe something like this?

Reply:
Yep I agree